service/resource/deployment/master_deployment.go

```diff
@@ -124,37 +124,16 @@ func newMasterDeployments(customObject kvmtpr.CustomObject) ([]*extensionsv1.Dep
 								Image:           customObject.Spec.KVM.EndpointUpdater.Docker.Image,
 								ImagePullPolicy: apiv1.PullIfNotPresent,
 								Command: []string{
-									"/bin/sh",
-									"-c",
-									"/opt/k8s-endpoint-updater update --provider.bridge.name=${NETWORK_BRIDGE_NAME} --provider.kind=bridge --service.kubernetes.address=\"\" --service.kubernetes.cluster.namespace=${POD_NAMESPACE} --service.kubernetes.cluster.service=master --service.kubernetes.inCluster=true --updater.pod.names=${POD_NAME}; while true; do sleep 1000000; done",
+									"/opt/k8s-endpoint-updater",
+									"update",
+									"--provider.bridge.name=" + key.NetworkBridgeName(key.ClusterID(customObject)),
+									"--service.kubernetes.cluster.namespace=" + key.ClusterNamespace(customObject),
+									"--service.kubernetes.cluster.service=" + MasterID,
+									"--service.kubernetes.inCluster=true",
 								},
```

Comment:
Before the actual command being executed got wrapped by a shell script. This did not forward process signals for termination. Now we get rid of the script wrapping. The actual go program receives termination signals and blocks. So we also do not need the while loops of the while loops anymore. Note that we also do not need to inject the pod name anymore because the bridge provider being used within the endpoint updater does not need it. See also https://github.com/giantswarm/k8s-endpoint-updater/pull/12.